### PR TITLE
Feat/recent news screen

### PR DESCRIPTION
**Pull request by victoria-illina**

### CODE REVIEW

File: css/news.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+.news__item-title {
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 24px;
+  color: #474747;
+  padding-left: 29px;
+}
+
+.news__item-title:hover {
```
Comment (andriy-ilin) [inline css/news.css]: you need to change text color, not even you move the cursor to block `title`,
change text color on when you move the cursor anywhere in block `item`

try something like this
`.news__item-title:hover {` => `. news__item:hover news__item-title {`

File: index.html [hunk 2]
```diff
@@ -320,5 +321,113 @@ <h3 class="pricing__item-title">Premium</h3>
...
+            </p>
+            <p class="news__item-text">
+              Ut enim ad minim veniquis nostrud lieon tear exercitation
+              ullamco.If you use this site to donating a bandwidth bill.
+            </p>
+            <div class="news__item-link">
+              <p class="news__item-link-text">Read More</p>
+              <div class="news__item-link-svg">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
```
Comment (andriy-ilin) [inline index.html]: you can remove this string

File: index.html [hunk 3]
```diff
@@ -320,5 +321,113 @@ <h3 class="pricing__item-title">Premium</h3>
...
```
Comment (andriy-ilin) [inline index.html]: remove `xmlns="http://www.w3.org/2000/svg"` nothing change

File: index.html [hunk 4]
```diff
@@ -320,5 +321,113 @@ <h3 class="pricing__item-title">Premium</h3>
...
```
Comment (andriy-ilin) [inline index.html]: remove xmlns="http://www.w3.org/2000/svg" nothing change

File: index.html [hunk 5]
```diff
@@ -320,5 +321,113 @@ <h3 class="pricing__item-title">Premium</h3>
...
+                  width="9"
+                  height="16"
+                  viewBox="0 0 9 16"
+                >
+                  <g>
+                    <g>
```
Comment (andriy-ilin) [inline index.html]: remove `<g>
                    <g>`

and `</g>
                  </g>`

on svg.

`g` in svg like empty `div` you can delete this tags

**Comment by victoria-illina** [comment]
#11

**Pull request merged**